### ISSUE TITLE
[Enhancement]: multiple managers

### ISSUE THREAD

**Issue by cole007** [olivierbon/craft-concierge#3]
It would be great if multiple managers could be specified for new registration notifications, or even better this be specified on a per-user or per-group basis (eg via the user and/or group settings)

**Comment by noregt** [comment]
Agree, this is exactly what I am missing, even better would be the ability to link this to a user field.

**Comment by olivierbon** [comment]
I'll work onto multiple comma separated recipients to start with. Will let you know about the rest when I look at it closer.

**Comment by outline4** [comment]
+1
